### ISSUE TITLE
The Hero Editor tutorial Plnkr example is broken.

### ISSUE THREAD

**Issue by owensgit** [angular/angular#18177]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior
In the Angular.io Hero Editor tutorial, if you click on the live example link, it opens a broken Plnkr page. The preview window displays "Loading..." and never loads the rest of the app.

Live example link: https://angular.io/generated/live-examples/toh-pt1/eplnkr.html

## Expected behavior
The live example should load with no problems.

## Minimal reproduction of the problem with instructions
*STEPS TO REPRODUCE*
1. Go to part one of the [Hero Editor tutorial](https://angular.io/tutorial/toh-pt1) in the Angular.io docs.
2. Scroll down just past the file structure.
3. See the sentence: When you're done with this page, the app should look like this live example / download example.
4. Click on live example.

## What is the motivation / use case for changing the behavior?
The motivation is that people new to Angular will be able to see the live example and follow the tutorial with confidence.

## Environment

<pre><code>
Angular version: Not known.

Browser:
- [x] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: n/a
- Platform:  Mac

**Comment by owensgit** [comment]
@alxhub - Hi there, do you know if this is something that will get looked at soon? This Plnkr example is important for newcomers to Angular and may put them off if they see it doesn't work.

**Comment by gkalpak** [comment]
I just tried it and it works fine for me. Are you getting errors in the console? Or do you have an extension that might be interferring with it (e.g. an ad-blocker)?

**Comment by owensgit** [comment]
@gkalpak thanks for testing it - it was my ad blocker that was breaking it! Apologies for the false alarm. Perhaps it would be useful to include a small message asking users to turn off their ad blocker in the documentation underneath the link? I can imagine lots of people who are new to Angular and Plnkr getting caught out like this.

**Comment by gkalpak** [comment]
I wouldn't be too keen on asking users to disable their ad-blockers just to see an example. It would be better if we could identify and fix the underlying issue.

Were you able to pinpoint what exactly triggered the ad-blocker to block the example?

**Comment by fegumi2003** [comment]
it's also not working for me.

**Comment by aravindfz** [comment]
@fegumi2003  is it working for you? 
@gkalpak can we take this to closure!

**Comment by gkalpak** [comment]
I am going to close since it is working as expected (ad-blockers aside) and we didn't get any specific feedback on why ad-blockers might be blocking it, so this issue is not actionable in its current state.

If there is a specific change that you find makes ad-blockers happier, please open a new issue or pull request.

**Comment by RoestVrijStaal** [comment]
@gkalpak 
You're wrong. 

It doesn't even work when uBlock (and similair is disabled). I get a `Error: Uncaught (in promise): [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: https://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js :: BrowserPlatformLocation.prototype.replaceState :: line 1811"  data: no] ...` error which don't give any clue thanks to the crappy zone.js.

relevant: https://github.com/angular/material2/issues/7573

tl;dr:  Do NOT use Plunker anymore! StackBlitz is the way to go.

**Comment by sarunint** [comment]
@RoestVrijStaal It works fine for me.

If it does not work for you, could you please provide the link to the Plunker that doesn't work, and tell us more about your browser? May be your internet blocks https://unpkg.com?

By the way, https://angular.io is currently in transition phase from Plunker to Stackblitz.
